### ISSUE TITLE
Issue with mappings.wasm

### ISSUE THREAD

**Issue by RichiCoder1** [vercel/ncc#687]
After upgrading to the `0.28.1`, I'm now seeing the following error:
```
> example-action@0.0.1 build
> ncc build -m -s index.ts

ncc: Version 0.28.1
ncc: Compiling file index.js
ncc: Using typescript@4.2.3 (local user-provided)
[Error: ENOENT: no such file or directory, open '/home/runner/work/actions/actions/node_modules/@vercel/ncc/dist/ncc/mappings.wasm'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/runner/work/actions/actions/node_modules/@vercel/ncc/dist/ncc/mappings.wasm'
}
npm ERR! Lifecycle script `build` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: example-action@0.0.1 
npm ERR!   at location: /home/runner/work/actions/actions/example-action 
```

----
Node Version: v14.16.0
Npm Version: 7.8.0

Using `npm workspaces` feature, this build is running in a workspace.

**Comment by styfle** [comment]
Thanks! This will be fixed in #688 👍

**Comment by RichiCoder1** [comment]
That fixed it! Thanks for jumping on this so quickly!
